user/src/assets/styles/globals.css

```
@@ -0,0 +1,72 @@
+@import "tailwindcss";
+@import "./variables.css";
+
+@theme {
```

Stylelint (17.12.0)

[error] Unexpected unknown at-rule "`@theme`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)